### ISSUE TITLE
White surfaces on components

### ISSUE THREAD

**Issue by Thunear** [digdir/designsystemet#2845]
We need white surfaces on our components. Using background-default for this does not fit the pattern and introduces issues when we try to create logic for surfaces.

* Test neutral colors with alpha
* Test alpha blending and WCAG contrasts

**Comment by Thunear** [comment]
**Meeting 17/12:** @mimarz @mrosvik @Febakke @Thunear @eirikbacker + 2 extra members from Mattilsynet
* We decided on making Surface Default in light mode white, and make the transition to Surface hover- and active states less noticeable. This way the grey color will work better on colored backgrounds.
* For neutral in dark mode we decided on using alpha values. This might work better since surfaces are never completely black.
* If the user wants grey surfaces, this color has to be added manually.

I will test this new approach and call for a new meeting a bit later.

**Comment by Thunear** [comment]
**Update 13:01:**
Tested making a neutral scale with alpha values to solve the issue with a white surface. After many different attempts the conclusion is that is does not work. There are too many requirements to fit into a singe scale. Using alpha values is also very problematic depending on the background layer.

**Possible solution:**
Will be testing adding the white surface color into the existing color scale. This is basically how the Background Default color works.

**Comment by Thunear** [comment]
In conlusion, this is what we will do:
* Rename `Surface Default` to `Surface Tinted`
* Add a new Surface color called `Surface default`
* Rename `Background subtle` to `Background Tinted`

**Comment by mrosvik** [comment]
> In conlusion, this is what we will do:
> 
> * Rename `Surface Default` to `Surface Tinted`
> * Add a new Surface color called `Surface default`
> * Rename `Background subtle` to `Background Tinted`

This will be done in #3029
